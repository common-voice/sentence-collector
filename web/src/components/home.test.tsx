import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

import Home from './home';

test('should render home', () => {
  const { container } = render(
    <BrowserRouter>
      <Home />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
