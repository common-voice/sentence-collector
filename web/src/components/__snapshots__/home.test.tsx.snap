// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render home 1`] = `
<div>
  <div
    id="home"
  >
    <h1>
      Welcome to the Common Voice Sentence Collector
    </h1>
    <p>
      The Sentence Collector is part of
       
      <a
        href="https://commonvoice.mozilla.org/"
      >
        Common Voice
      </a>
      . It allows contributors to collect and validate sentences created by the community. You can use this tool also to import and clean-up small-to-medium-sized public domain corpus you have found or collected. All sentences need to be Public Domain. Approved sentences are exported every week to the Common Voice repository and are released on the Common Voice website on every new deployment.
    </p>
    <section
      id="contribute"
    >
      <a
        href="/add"
      >
        <div
          class="contributeCard"
        >
          <h2>
            Collect sentences
          </h2>
          <p>
            Help us by writing or collecting Public Domain sentences.
          </p>
        </div>
      </a>
      <a
        href="/review"
      >
        <div
          class="contributeCard"
        >
          <h2>
            Review sentences
          </h2>
          <p>
            Help us by reviewing sentences for correctness according to the guidelines.
          </p>
        </div>
      </a>
    </section>
  </div>
</div>
`;
